### PR TITLE
Syntax highlight

### PR DESCRIPTION
**Pull request by deeechun**
I've added in a source pointing to a link online for the highlighter.js file. If you want, I can download the js file locally and just point to the file in case the site i'm pointing to goes down

### CODE REVIEW

File: _includes/js.html [hunk 1]
```diff
@@ -15,3 +15,10 @@
 
     <!-- Custom Theme JavaScript -->
     <script src="{{ "/js/agency.js" | prepend: site.baseurl }}"></script>
+
+     <!-- Syntax Highlighter JavaScript -->
+    <link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/default.min.css">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js">
+    </script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/languages/gherkin.min.js">
+    </script>
```
Comment (timofurrer) [inline _includes/js.html]: You have to enable highlightjs in order to make it work. Add the following after including the JS files from the CDN:

```html
<script>hljs.initHighlightingOnLoad();</script>
```

File: _includes/gettingstarted.html [hunk 2]
```diff
@@ -62,7 +65,9 @@ <h4 class="subheading">Write our steps</h4>
                                 </div>
                                 <div class="timeline-body">
                                     <p class="text-muted">Every line inside a Scenario is called a step. You have to provide a python function for every step in order to execute some code. Let's create a new "radish" directory and write our first step.py file there:
-                                        <pre style="width: 500px; text-align: left; margin-left: -100px"># -*- coding: utf-8 -*-
+                                        <pre style="width: 500px; text-align: left; margin-left: -100px">
+                                            <code class="Python">
+                                            # -*- coding: utf-8 -*-
```
Comment (timofurrer) [inline _includes/gettingstarted.html]: Inside `<code>`-tags the indentation actually matters. That's why this line will be indented in a weird way. 
Also pay attention to leading and trailing new/blank lines.

**Review by timofurrer — CHANGES_REQUESTED** [review]
Thanks for the PR! :tada: 

You can actually have a look at how it looks generated by serving the site with jekyll:

```bash
# you might want to run this as root
gem install jekyll bundler
# serve the site
jekyll serve
```

:beers:

**Comment by deeechun** [comment]
Wow you're a life saver. I was wondering how I can check changes online for indentation errors, etc. 

Also, I completely forgot to add in that initialization script after playing around with it for a little bit XD. I will fix the indentation and submit another PR.

**Comment by timofurrer** [comment]
Looks much better now. Is it possible to remove this gray border around the code blocks?

image

**Comment by deeechun** [comment]
Does this look alright?
screenshot from 2017-03-02 13-06-35

**Comment by deeechun** [comment]
Would this be alright?
screenshot from 2017-03-02 13-08-02

**Comment by timofurrer** [comment]
Yeah, looks better. Can we make a 1px border though? Like before just without the cap? And please increase the width of the steps code block a little so the `given` decorator fits on one line.

If some of those is not possible let me know.

**Comment by deeechun** [comment]
Sure, changed it. I'll commit now.
screenshot from 2017-03-02 15-04-37

**Review by timofurrer — APPROVED** [review]

**Comment by timofurrer** [comment]
Awesome, thanks! :beers:

**Pull request merged**